README.md

```diff
@@ -24,21 +24,38 @@ To run that demo on your own computer:
 * `npm run storybook`
 * Visit http://localhost:9001/
 
-## How to Use
+## Getting Started
+#### Install dependencies
+```
+npm install --save react-dates
+```
+Make sure you have the following dependencies installed:
+```
+ npm install --save moment react react-dom react-addons-shallow-compare
```

Comment:
oops some of my message got cut off... trace is below - it looks like type: inspectdata is throwing it off  (as I imagine the final type statement will also cause errors)... not sure if this is standard output or if it's because I'm a naive Cygwin user and I'm doing something wrong - if it is standard output I'm guessing npm just ignores the type stuff?

```
Trace: 
  Error: type:inspectdata/:moment:2.10-2.14%7C%7C%5E2.15.1react:%3E=0.14react-dom:%3E=0.14react-addons-shallow-compare:%3E=0.14: Invalid protocol: type:
      at Request.init (C:\Program Files (x86)\Yarn\node_modules\request\request.js:460:31)
      at new Request (C:\Program Files (x86)\Yarn\node_modules\request\request.js:129:8)
      at request (C:\Program Files (x86)\Yarn\node_modules\request\index.js:55:10)
      at RequestManager.execute (C:\Program Files (x86)\Yarn\lib\util\request-manager.js:327:17)
      at RequestManager.shiftQueue (C:\Program Files (x86)\Yarn\lib\util\request-manager.js:353:10)
      at data (C:\Program Files (x86)\Yarn\lib\util\request-manager.js:251:12)
      at Request.params.callback [as _callback] (C:\Program Files (x86)\Yarn\lib\util\request-manager.js:305:11)
      at Request.self.callback (C:\Program Files (x86)\Yarn\node_modules\request\request.js:187:22)
      at emitTwo (events.js:106:13)
      at Request.emit (events.js:191:7)
```